services/autoscaling/src/simcore_service_autoscaling/core/settings.py

```diff
@@ -14,6 +14,15 @@
 from .._meta import API_VERSION, API_VTAG, APP_NAME
 
 
+class AwsSettings(BaseCustomSettings):
```

Comment:
aws region? not sure if this needs to go here, ignore if not applicable

Reply:
I took this verbatim from @Surfict script. See where is used.
In your opinion, what settings would be appropriate here?

Reply:
I added aws-region.

Reply:
I think all the env from this file : https://git.speag.com/allexandre/osparc-dask-auto-scaling/-/blob/master/.env are necessary. Aws region is not necessary as AWS know it from the subnet_id already.

Reply:
Reading the code again, aws_region is necessary.